docs/libpod.conf.5.md

```diff
@@ -24,6 +24,9 @@ libpod to manage containers.
 **cgroup_manager**=""
   Specify the CGroup Manager to use; valid values are "systemd" and "cgroupfs"
 
+**init_path**=""
+  Path to the container-init binary, which forwards signals and reaps processes within containers.
```

Comment:
Indicate this is only used if --init is set.

Reply:
Done